### DISCUSSION TITLE
ESM Support?

### DISCUSSION THREAD

**Discussion by davidhousedev** [sergiodxa/remix-auth#283] [General]
While the source code for remix-auth is written using ESM, the built package only supports CJS. When using `remix-auth` in a Vite-based Remix app, this requires `remix-auth` to be excluded from native ESM support via `ssr.optimizeDeps.include`. Do you have any plans to support a build of the package that supports an ESM entrypoint?

Thanks so much!

**Comment by sergiodxa** [comment]
Eventually I plan to do the change to ESM, but since that's a breaking change I have been delaying it, specially when it's possible to use it right now being CJS.
